components/dialogs/MissionDialog.vue

```diff
@@ -367,7 +367,8 @@ export default {
     },
     subDescription() {
       if (
-        !this.shouldShowDesktopOnly
+        !this.mission.isExpired
+        && !this.shouldShowDesktopOnly
         && this.hasReferrer
         && this.missionId === 'joinTokenSale'
```

Comment:
should `isExpired` and `shouldShowDesktopOnly` outside of `joinTokenSale` related checking?

Reply:
Updated, and i think `shouldShowDesktopOnly` is not relevant to `joinTokenSale` as well as `joinTokenSale` is an expired mission so i removed it